rs/cli/src/commands/api_boundary_nodes/add.rs

```diff
@@ -23,7 +23,7 @@ pub struct Add {
 
 impl ExecutableCommand for Add {
     fn require_ic_admin(&self) -> IcAdminRequirement {
-        IcAdminRequirement::Detect
+        IcAdminRequirement::default()
```

Comment:
What is default? This means I have to go the specific code to see which thing is default. Also the default can change and all the commands would change the requirement. I would prefer being explicit rather than having language magic for configuration of commands